cassandane/Cassandane/Instance.pm

```diff
@@ -1412,6 +1412,56 @@ sub _check_valgrind_logs
     return;
 }
 
+sub _sanitizer_log_dir()
+{
+    my ($self, $sanitizer) = @_;
+
+    my $san_logdir = $self->{basedir} . "/${sanitizer}logs/";
+    mkpath $san_logdir
+        unless ( -d $san_logdir );
+
+    return $san_logdir;
+}
+
+sub _check_sanitizer_logs
+{
+    my ($self, $sanitizer) = @_;
+
+    my $san_logdir = $self->_sanitizer_log_dir($sanitizer);
+
+    opendir my $dirfh, $san_logdir
+        or return "Cannot open directory $san_logdir for reading: $!";
+
+    my @nzlogs;
+    while ($_ = readdir $dirfh)
+    {
+        next if m/^\./;
+        next if m/\.core\./;
```

Comment:
The core files pattern is configurable in cassandane.ini, so maybe use that here too, see `sub find_cores` for an example of it in use

Reply:
Righto, we should have an issue for this - I stole this from the valgrind code, so we should fix that elsewhere